Remotion/Web/Development.WebTesting/WebDriver/Configuration/Chrome/ChromiumDisableSecurityWarningsBehavior.cs

```diff
@@ -0,0 +1,43 @@
+﻿// This file is part of the re-motion Core Framework (www.re-motion.org)
+// Copyright (c) rubicon IT GmbH, www.rubicon.eu
+// 
+// The re-motion Core Framework is free software; you can redistribute it 
+// and/or modify it under the terms of the GNU Lesser General Public License 
+// as published by the Free Software Foundation; either version 2.1 of the 
+// License, or (at your option) any later version.
+// 
+// re-motion is distributed in the hope that it will be useful, 
+// but WITHOUT ANY WARRANTY; without even the implied warranty of 
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the 
+// GNU Lesser General Public License for more details.
+// 
+// You should have received a copy of the GNU Lesser General Public License
+// along with re-motion; if not, see http://www.gnu.org/licenses.
+// 
+using System;
+
+namespace Remotion.Web.Development.WebTesting.WebDriver.Configuration.Chrome
+{
+  /// <summary>
+  /// Specifies the possible behaviors related to the <c>CommandLineFlagSecurityWarningsEnabled</c> registry flag responsible for hiding infobars
+  /// showing "Chrome is being controlled by automated test software".
+  /// </summary>
+  public enum ChromiumDisableSecurityWarningsBehavior

```

Comment:
ChromeDisableSecurityWarningsBehavior

Reply:
@MichaelKetting  This will be a setting needed for Edge also, that's why I renamed it back.

Reply:
okay, we know this will work in both browsers the same way?
In this case, the setting should not be in the chrome folder. and the configsection should be named chromiumConfigsection and used twice in the config, once for chrome and once for edge.

Reply:
Yes it works the same way in both browsers.